scripts/tunnel.ps1

```diff
@@ -1,5 +1,12 @@
 [CmdletBinding()]
-param()
+param(
+    [parameter(
+        Mandatory = $true, HelpMessage="Burp Collaborator address")]
+    $collabDomain,
```

Comment:
Do we want to make this line and line 7 non-mandatory? My powershell is rusty, but can they run this currently without providing the arguments?

Reply:
Right, the script currently prompts the user to ask for these values. It's counter-intuitive, but if the params are marked as "Mandatory = $true" and they aren't provided, then powershell will automatically prompt the user to provide them. So by marking these as mandatory, that's how we maintain the existing behavior.

Reply:
Ahh, sweet. Nice!